R/leaflet.R

```diff
@@ -70,3 +102,71 @@ mapOptions <- function(map, zoomToLimits = c("always", "first", "never")) {
 
   map
 }
+
+# CRS classes supported
+crsClasses <- list( 'L.CRS.EPSG3857', 'L.CRS.EPSG4326', 'L.CRS.EPSG3395',
+                    'L.CRS.Simple', 'L.Proj.CRS', 'L.Proj.CRS.TMS' )
```

Comment:
Just checking--we'll be able to support (at least) these CRS classes even after we move to Leaflet 1.0?